adapters/saml/wildfly/wildfly-jakarta-subsystem/pom.xml

```diff
@@ -140,6 +140,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
@fjuma , does this fix requires this new dependency with test scope?

Reply:
@jfdenise This new dependency was needed to get the Keycloak subsystem parsing tests to pass. It's not related to the underlying problem here. It seemed to be a separate issue that was causing the parsing tests to fail.